release-notes/release_notes-2.18.0.md

```diff
@@ -0,0 +1,800 @@
+# OpenSearch and OpenSearch Dashboards 2.18.0 Release Notes
+
+
+## FEATURES
+
+
+### Dashboards Assistant
+
+
+* feat: Add new feature to support text to visualization([#264](https://github.com/opensearch-project/dashboards-assistant/pull/264),[#299](https://github.com/opensearch-project/dashboards-assistant/pull/299),[#310](https://github.com/opensearch-project/dashboards-assistant/pull/310),[#313](https://github.com/opensearch-project/dashboards-assistant/pull/313),[#325](https://github.com/opensearch-project/dashboards-assistant/pull/325),[#327](https://github.com/opensearch-project/dashboards-assistant/pull/327),[#330](https://github.com/opensearch-project/dashboards-assistant/pull/330),[#312](https://github.com/opensearch-project/dashboards-assistant/pull/312),[#350](https://github.com/opensearch-project/dashboards-assistant/pull/350),[#354](https://github.com/opensearch-project/dashboards-assistant/pull/354),[#351](https://github.com/opensearch-project/dashboards-assistant/pull/351))
```

Comment:
Is it possible to remove all `feat:` wordings? Just keep the main PR title?

Reply:
I am ok to keep feat, but just need to capitalize all the first word of each line.

Reply:
Agree with @peterzhuamazon.